Models/tournaments/mappools/mappool.ts

```diff
@@ -26,6 +26,15 @@ export class Mappool extends BaseEntity {
     @Column({ default: false })
     isPublic!: boolean;
 
+    @Column({ type: "varchar", nullable: true })
+    s3Key?: string | null;
```

Comment:
useless to store, implement a getter that parses the link and return the path instead